test/matchAnyPCat.html

```diff
@@ -0,0 +1,44 @@
+<html><head>  
+    <title>SPOT Automated Testing</title>
+
+	<link rel="stylesheet" type="text/css" href="../spot.css">
+
+    <!--If running tests that do not have to do with the interface, run jsbuild.sh or jsbuild.bat
+        in root directory of the project. If running interface testing, run testbuild.sh
+        or testBuild.bat-->
+    <script src="../build/spot.js"></script>
+
+    <!--Mocha setup stuff (1/2)-->
+    <meta charset="UTF-8">
+    <script src="../lib/test/mocha.min.js"></script>
+    <script src="../lib/test/chai.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="../lib/test/mocha.css">
+    <!--mocha setup stuff to be continued in document body-->
+    
+</head>
+<body style="padding-left: 5%; padding-right: 5%; padding-top: 20px">
+
+    <h2>SPOT Automated Testing</h2>
+
+    <!--Import error message functions-->
+    <script src="./message_funcs.js"></script>
+
+    <!--Mocha stuff (2/2)-->
+    <script>
+        mocha.setup("bdd"); //brings "describe", "it", etc. into global namespace
+        //mocha.checkLeaks();
+        var assert = chai.assert; //no one wants to type out "chai.assert" every time
+    </script>
+    <div id="mocha">
+        Testing MatchSP 'any' category
+    </div>
+    <!--Mocha is set up now, you just need to write and run tests-->
+    <script src="matchAnyPCat.js"></script>
+    <script>
+            mocha.run(); //don't forget to run the test!!!
+    </script>
+    <!--That's all you need for testing with mocha and chai-->
+
```

Comment:
Add a tableau to this one too. Note that the call to writeTableau needs to go here in the html file, not in the .js file with the actual test function. But you could run makeTableau() in the test function, save its return value to a variable, and return that in this file so that you don't have to make trees and so on in the html file itself.